.github/workflows/vib-verify.yaml

```diff
@@ -16,11 +21,20 @@ jobs:
       chart: ${{ steps.get-chart.outputs.chart }}
       result: ${{ steps.get-chart.outputs.result }}
     steps:
+      # Using actions/github-scripts because using exit 1 in the script above would not provide any output
+      # Source: https://github.community/t/no-output-on-process-completed-with-exit-code-1/123821/3
+      - id: check-approval
+        name: 'Check approval'
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'verify') }}
+        uses: actions/github-script@v3
+        with:
+          script: |
+              core.setFailed('Approval to run this job is required')
```

Comment:
Maybe we can add something like this:
`Internal approval to run this job is required, please contact someone from the organisation.`
For UX purposes only.

Reply:
👍